enterprise/backend/src/metabase_enterprise/serialization/v2/load.clj

```diff
@@ -107,6 +108,18 @@
           ctx      {:expanding #{}
                     :seen      #{}
                     :ingestion ingestion
-                    :from-ids  (m/index-by :id contents)}]
+                    :from-ids  (m/index-by :id contents)
+                    :errors    []}]
       (log/infof "Starting deserialization, total %s documents" (count contents))
-      (reduce load-one! ctx contents))))
+      (reduce (fn [ctx item]
+                (try
+                  (load-one! ctx item)
+                  (catch Exception e
+                    (if skip-errors
+                      (do
+                        ;; eschew big and scary stacktrace
+                        (log/warnf "Skipping deserialization error: %s %s" (ex-message e) (ex-data e))
+                        (update ctx :errors conj e))
+                      (throw e)))))
```

Comment:
Not sure which I prefer actually, but you can save a `do` and two layers of nesting if you re-throw first.

```suggestion
                    (when-not skip-errors
                      (throw e))
                    ;; eschew big and scary stacktrace
                    (log/warnf "Skipping deserialization error: %s %s" (ex-message e) (ex-data e))
                    (update ctx :errors conj e))))
```

Reply:
Don't trust this diff, I didn't count closing braces carefully.